MAINTAINERS.md

```diff
@@ -0,0 +1,128 @@
+# Maintainers
+
+## Maintainer Scopes, GitHub Roles and GitHub Teams
+
+Maintainers are assigned the following scopes in this repository:
+
+| Scope      | Definition            | GitHub Role | GitHub Team                             |
+| ---------- | --------------------- | ----------- | --------------------------------------- |
+| Admin      | The GitHub Admin Role | Admin       | [aries-admins]                          |
+| Write      | The GitHub Write role | Write       | [aries-cloudagent-python committers]    |
+| Maintainer | The GitHub Read role  |             | [aries-socketdock-committers]           |
+| Read       | The GitHub Write role | Write       | [aries-framework-javascript committers] |
```

Comment:
I'm a bit new to the whole scope thing, but these two scopes appear to need a little work. For example: I would think that the Maintainer scope would have a little more access than just read access

Reply:
A typo — thanks. Fixed!